libcst/codemod/visitors/tests/test_apply_type_annotations.py

```diff
@@ -292,6 +292,23 @@ def foo() -> bar.Baz:
                     return returns_baz()
                 """,
             ),
+            "with_as_import": (
+                """
+                def foo(x): ...
+                """,
```

Comment:
In the test cases, the stub comes first, then the input code, then the output code.

So I don't think this test case is verifying what you are hoping for - here we're just not applying a stub with no annotations.

Reply:
oops, no wonder it was still passing when i messed up a rebase :) fixed both now.